### ISSUE TITLE
#0001 | Calling a file that does not exist. 

### ISSUE THREAD

**Issue by ntodzy** [Draekdude/BoyScoutWreathTracker#1]
\BoyScoutWreathTracker-master\BoyScoutWreathTracker\BoyScoutWreathTracker\MainWindow.xaml.cs

Instanced Line 52-58

`
private void loadXMLFiles(WreathData wreathData)
        {
            //wreathData.Wreath.ReadXml(wreathPath);
            //wreathData.Scout.ReadXml(scoutPath);
            wreathData.Inventory.ReadXml(inventoryPath); 
            wreathData.Payments.ReadXml(paymentsPath);
        }

`
Issue Line 25-28
`
 const string wreathPath = "C:\\Boy Scouts\\wreath.xml";
        const string scoutPath = "C:\\Boy Scouts\\scout.xml";
        const string inventoryPath = "C:\\Boy Scouts\\inventory.xml";
        const string paymentsPath = "C:\\Boy Scouts\\payments.xml";
`

TLDR: call relative to the installation folder.
